assets/script/build_app.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# set env vars
+GOPATH=$HOME/gocode
+GOROOT=$HOME/go
+PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+
+if [ "$(uname)" == "Darwin" ]; then
+  npm run electron-pack
+else
+  # build binaries for windows
+  cd assets/bin/win32
+  env GOOS="windows" GOARCH="386" go build -v github.com/lightningnetwork/lnd
+  env GOOS="windows" GOARCH="386" go build -v github.com/btcsuite/btcd
+
+  # build the packages using electron-builder on docker
+  cd $TRAVIS_BUILD_DIR
+  env | grep -iE 'DEBUG|NODE_|ELECTRON_|YARN_|NPM_|CI|CIRCLE|TRAVIS_TAG|TRAVIS|TRAVIS_REPO_|TRAVIS_BUILD_|TRAVIS_BRANCH|TRAVIS_PULL_REQUEST_|APPVEYOR_|CSC_|GH_|GITHUB_|BT_|AWS_|STRIP|BUILD_' > env.txt
+  docker run --rm \
```

Comment:
awesome, i'll test the resulting windows and linux packages on my machine to make sure they work

Reply:
@valentinewallace I've been testing this and finally got the build working for all platforms again. Testing auto-update now.

Reply:
ah you're able to test builds for non-mac? ok sounds good :)

Reply:
Yeah I just trigger builds on travis by tagging a git commit :)